critiquebrainz/frontend/views/artist.py

```diff
@@ -30,6 +33,16 @@ def entity(mbid):
     if request.args.get('reviews') == "all":
         artist_reviews_limit = None
 
+    if current_user.is_authenticated:
+        my_reviews, my_count = db_review.list_reviews(
+            entity_id=artist['id'],
+            entity_type='artist',
+            user_id=current_user.id,
+        )
+        my_review = my_reviews[0] if my_count else None
+    else:
+        my_review = None
+
```

Comment:
You can also set `my_review = None` before checking user authentication, and update only if there is a review for the authenticated user.